application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -50,8 +50,9 @@ public final class HelpThreadCreatedListener extends ListenerAdapter
      *
      * @param helper to work with the help threads
      */
-    public HelpThreadCreatedListener(HelpSystemHelper helper) {
+    public HelpThreadCreatedListener(HelpSystemHelper helper, JDA jda) {
```

Comment:
its unecessary for u to memorize JDA, as u can get hands on the JDA instance on pretty much any jda class already.
i.e. in ur code below, just do `threadChannel.getJda()` and u have it